packages/backend/src/app/workers/sandbox.ts

```diff
@@ -22,33 +23,65 @@ export class Sandbox {
     private static readonly isolateExecutableName = getIsolateExecutableName()
     private static readonly sandboxRunTimeSeconds = system.getNumber(SystemProp.SANDBOX_RUN_TIME_SECONDS) ?? TWO_MINUTES
 
-    constructor(public readonly boxId: number) {}
+    public readonly boxId: number
+    public used: boolean
+    public cached: boolean
+    public resourceId: string
+    public lastUsed: number
 
-    async cleanAndInit(): Promise<void> {
+    constructor(request: { boxId: number, used: boolean, resourceId: string, lastUsed: number, cached: boolean }) {
+        this.boxId = request.boxId
+        this.used = request.used
+        this.cached = request.cached
+        this.resourceId = request.resourceId
+        this.lastUsed = request.lastUsed
+    }
+
+
+    async recreate(): Promise<void> {
         await Sandbox.runIsolate('--box-id=' + this.boxId + ' --cleanup')
         await Sandbox.runIsolate('--box-id=' + this.boxId + ' --init')
         await packageManager.initProject(this.getSandboxFolderPath())
     }
 
+    async clean(): Promise<void> {
+        const filesToDelete = [
+            '_functionOutput.txt',
+            '_standardOutput.txt',
+            '_standardError.txt',
+            'output.json',
+            'meta.txt',
+        ]
+        const promises = filesToDelete.map((file) => {
+            const filePath = path.join(__dirname, this.getSandboxFilePath(file))
+            return fs.unlink(filePath).catch((error) => {
```

Comment:
Any reason to use `fs#unlink` instead of the more obvious `fs#rm`? Also why use `Promise#catch` instead of a `try/catch` block?